bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/validation/zeebe/ZeebeElementValidator.java

```diff
@@ -92,7 +92,9 @@ private long countMatchingGroupAttributesAssertions(final T element) {
         .filter(
             assertion -> {
               final String attributeValue = assertion.attributeSupplier.apply(element);
-              return attributeValue != null && !attributeValue.isEmpty();
+              return attributeValue != null
+                  && !attributeValue.isEmpty()
+                  && !" ".equals(attributeValue);
```

Comment:
❌ We should also prevent sending strings that contains multiple spaces. It is wiser to use `!attributeValue.trim().isEmpty()` in Java 8.

Reply:
You're right, nice one 💪